vizro-core/docs/pages/user-guides/install.md

```diff
@@ -1,61 +1,146 @@
 
-- **Python**: Vizro supports macOS, Linux, and Windows. It is designed to work with Python 3.8 and above. The python
-  version can be specified when setting up your virtual environment.
-- **Virtual environment**: We suggest to create a new virtual environment for each new Vizro project you work on to
-  isolate its Python dependencies from those of other projects. See the following references to learn more about [python virtual environments](https://realpython.com/python-virtual-environments-a-primer/), [conda virtual environments](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#starting-conda) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
+**Python**: Vizro supports macOS, Linux, and Windows. It works with Python 3.8 and later. You can specify the version of Python to use with Vizro when you set up a virtual environment.
 
 
-??? tip "Beginners/Code novices"
-    If you consider yourself a beginner or coding novice, you can follow the steps in these boxes to completely avoid the terminal usage
+**Virtual environment**: You should create a virtual environment for each Vizro project you work on to
+  isolate its Python dependencies from those of other projects. See the following references to learn more about [Python virtual environments](https://realpython.com/python-virtual-environments-a-primer/), [Conda virtual environments](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#starting-conda) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
 
-    - install the [Anaconda Navigator](https://www.anaconda.com/download)
-    - create a new environment as [outlined here](https://docs.anaconda.com/free/navigator/tutorials/manage-environments/), choose a Python version `>=3.8`
+!!! tip "Prefer to use Vizro without opening a terminal window?"
 
-## Install
+    If you are a beginner or coding novice, you may prefer to avoid using a terminal to work with Vizro. Skip to the ["Use Vizro inside Anaconda" section](#use-vizro-inside-anaconda-navigator) below.
 
-To install Vizro from the Python Package Index (PyPI), use [`pip`](https://pip.pypa.io/en/stable/) in your terminal window:
+
+
+??? information "How to create a virtual environment for your Vizro project"
+
+    ### Using `conda`
+    We strongly recommend [installing `conda` as your virtual environment manager](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) if you don't already use it. Once installed, create a virtual environment from the terminal as follows:
+
+    ```bash
+    conda create --name vizro-environment python=3.10 -y
+    ```
+
+    (This example uses Python 3.10, and creates a virtual environment called `vizro-environment`. You can opt for a different version of Python (any version >= 3.8 and <3.12), and name it anything you choose).
+
+    Activate your virtual environment from any directory:
+
+    ```bash
+    conda activate kedro-environment
```

Comment:
Needs to match the one  in "conda create --name vizro-environment python=3.10 -y"

```suggestion
    conda activate vizro-environment
```